lib/tasks.js

```diff
@@ -130,8 +130,12 @@ const runTasks = async (opts, injected = {}) => {
       github.release && github.releaseNotes && logPreview(log, 'release notes', await ghClient.getNotes());
       const ghRelease = () => ghClient.release({ version, isPreRelease, changelog });
       const ghUploadAssets = () => ghClient.uploadAssets();
+      const ghPublishRelease = () => ghClient.maybePublishRelease();
       if (isInteractive) {
-        const release = () => ghRelease().then(() => ghUploadAssets());
+        const release = () =>
+          ghRelease()
+            .then(() => ghUploadAssets())
+            .then(() => ghPublishRelease());
```

Comment:
Please also update the flow for `isInteractive === false` below. This will then fix the `draft: true` assertions in `test/tasks.js` ;-)

The flows are slightly different. After prompted to release to GitHub, there should be a single `task` to complete. But in non-interactive mode, it's nice to separate the actions to better indicate the progress.